fimax/loans/doctype/insurance_card/insurance_card.js

```diff
@@ -66,8 +66,9 @@ frappe.ui.form.on('Insurance Card', {
 		frm.trigger("make_repayment_schedule");
 	},
 	"calculate_outstanding_amount": (frm) => {
-		if (frm.doc.initial_payment_amount > frm.doc.total_amount) {
+		if (frm.doc.initial_payment_amount >= frm.doc.total_amount) {
```

Comment:
Can you check on this line? It looks like your comparing *greater or equal than* and the message you're throwing out says something about *greater than* only